README.md

```diff
@@ -61,13 +61,47 @@ Create or delete a webhook using the configured parameters.
     -   `delete` to delete an existing webhook. Outputs current timestamp on non-existing webhooks.
 -   `events`: *Optional*. An array of [events](https://developer.github.com/webhooks/#events) which will trigger your webhook. Default: `push`
 
+## Example
+Include the github-webhook-resource in your pipeline.yml file
+
+```yaml
+resource_types:
+  - name: github-webhook-resource
+    type: docker-image
+    source:
+      repository: homedepottech/github-webhook-resource
+      tag: latest 
+```
+
+Now when you set your pipeline, you can optionally include instance variables that will be picked up by the resource. Here is a sample script that sets the pipeline for you. 
```

Comment:
I meant that you should add an example/description to the [Behavior](https://github.com/homedepot/github-webhook-resource#behavior) section specifically, but this example won't hurt